### PR TITLE
Update 94-appendixD.Rmd - LC9.8 and LC9.11

### PR DESCRIPTION
**Pull request by simplyjin**
LC9.8 - The larger significance level would lead to a more liberal hypothesis, not the smaller

LC9.11 - In chapter 9.5 we failed to reject the null hypothesis because the p-value of 0.004 was greater than our significance level of 0.001. Fixing the solution so it's in line with this.

### CODE REVIEW

File: 94-appendixD.Rmd [hunk 1]
```diff
@@ -1459,7 +1459,7 @@ The p-value's 0.05 threshold can be misleading researchers to conduct multiple b
 
 **Solution**:  
 
-The smaller $\alpha$ of 0.01 will lead to a more liberal hypothesis testing procedure, because the required p-value for reject the null hypothesis $H_0$ is smaller. 
+The greater $\alpha$ of 0.1 will lead to a more liberal hypothesis testing procedure, because the required p-value to reject the null hypothesis $H_0$ is greater. There is a higher chance that the p-value will be less than $\alpha$
```
Comment (rudeboybert) [inline 94-appendixD.Rmd:1462]:
```suggestion
The greater $\alpha$ of 0.1 will lead to a more liberal hypothesis testing procedure, because the required p-value to reject the null hypothesis $H_0$ can be greater. There is a higher chance that the p-value will be less than $\alpha$
```

**Review by rudeboybert — APPROVED** [review]

**Comment by rudeboybert** [comment]
Thanks @simplyjin!
